app/models/notification.rb

```diff
@@ -115,13 +115,13 @@ def remove_all_by_action_without_delay(notifiable_ids:, notifiable_type:, action
     def remove_all(notifiable_ids:, notifiable_type:)
       return unless %w[Article Comment Mention].include?(notifiable_type) && notifiable_ids.present?
 
-      Notifications::RemoveAllJob.perform_later(notifiable_ids, notifiable_type)
+      Notifications::RemoveAllWorker.perform_async(notifiable_ids, notifiable_type)
     end
 
     def remove_all_without_delay(notifiable_ids:, notifiable_type:)
       return unless %w[Article Comment Mention].include?(notifiable_type) && notifiable_ids.present?
 
-      Notifications::RemoveAllJob.perform_now(notifiable_ids, notifiable_type)
+      Notifications::RemoveAllWorker.perform_async(notifiable_ids, notifiable_type)
```

Comment:
Sorry, I totally missed this problem on the first pass: here it shouldn't queue the job, it should call directly the service:

```ruby
Notifications::RemoveAll.call(Array.wrap(notifiable_ids), notifiable_type)
```

Kudos to @serenaf and her comments in #5466 for making me notice this issue

Reply:
Ok. I will make the change. Sorry for not declare which jobs I was working on. Thanks for the advice :smile: